docs/src/usage.txt

```diff
@@ -11,13 +11,13 @@ Options:
       --edit-page            Edit custom page with `EDITOR`
       --edit-patch           Edit custom patch with `EDITOR`
   -f, --render <FILE>        Render a specific markdown file
-  -p, --platform <PLATFORM>  Override the operating system, can be specified multiple times in order
-                             of preference [possible values: linux, macos, sunos, windows, android,
-                             freebsd, netbsd, openbsd, common]
+  -p, --platform <PLATFORM>  Override the operating system, can be specified multiple times in order of preference [possible values: linux, macos, sunos, windows, android, freebsd,
```

Comment:
Here CI fails because we are comparing with the usage printed when the terminal is (I think) 80 chars wide. You can just edit this by hand according to the output of our CI output